PiwikTracker/Context.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+public class Context {
+    
+    public class Application {
+        // TODO: implement me
+    }
+    
+    public class Device {
+        
+        /// The platform name of the device i.e. "iPhone1,1" or "iPad3,6"
+        static let platform: String = {
```

Comment:
Using static methods make access to members more easy without passing around an instance.
However it is a bit like having singletons and might bite back if you ever need to mock the class in a unit test.

But I do not think that will be an issue here, so I would keep as is and refactor later the need arise.